src/test/java/core/basesyntax/dao/FruitShopDaoImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitShopDaoImplTest {
+    private static final String BANANA_KEY = "banana";
+    private static final String APPLE_KEY = "apple";
+    private static final int PUT_VALUE = 11;
```

Comment:
remove this constant and use its value directly in the test method where you need it